test/vcf/record_test.cpp

```diff
@@ -510,5 +510,58 @@ namespace ebi
                                 source}),
                             vcf::IdBodyError*);
         }
+
+        SECTION("Duplicate FILTERs")
+        {
+            CHECK_THROWS_AS( (vcf::Record{
+                                1,
+                                "chr1",
+                                123456,
+                                { "id123" },
+                                "A",
+                                { "AC" },
+                                1.0,
+                                { "q10", "q10" },
+                                { {"AN", "12"} },
+                                { "GT" },
+                                { "0|1" },
+                                source}),
+                            vcf::FilterBodyError*);
+        }
+
+        SECTION("FILTER with value 0")
+        {
+            CHECK_THROWS_AS( (vcf::Record{
+                                1,
+                                "chr1",
+                                123456,
+                                { "id123" },
+                                "A",
+                                { "AC" },
+                                1.0,
+                                { "q1", "0" },
+                                { {"AN", "12"} },
+                                { "GT" },
+                                { "0|1" },
+                                source}),
+                            vcf::FilterBodyError*);
+        }
+        SECTION("Duplicate INFOs") 
+        {
+            CHECK_THROWS_AS( (vcf::Record{
+                                1,
+                                "chr1", 
+                                123456, 
+                                { "id123", "id456" }, 
+                                "A",
+                                { "T", "C" },
+                                1.0, 
+                                { "PASS" }, 
+                                { {"AN", "12"}, {"AN", "15"} }, 
+                                { "DP" }, 
+                                { "12" },
+                                source}),
+                            vcf::InfoBodyError*);
+        }
```

Comment:
Could `SECTION("Duplicate FORMATs")` be placed after INFO to follow the same order as the VCF format? It could make a bit easier to find where the tests are.